configure.ml

```diff
@@ -647,8 +647,11 @@ let camltag = match caml_version_list with
     48: implicit elimination of optional arguments: too common
     50: unexpected documentation comment: too common and annoying to avoid
     56: unreachable match case: the [_ -> .] syntax doesn't exist in 4.02.3
+    58: "no cmx file was found in path": See https://github.com/ocaml/num/issues/9
+    59: "A potential assignment to a non-mutable value was detected in this source file.":
+        See Coq's issue #8043
```

Comment:
It would be nicer it this fit in 1 line, maybe
`59: potential assignment to a non-mutable value, see #8043`

Reply:
Fixed.